src/compiler/reviews.md

```diff
@@ -1,4 +1,156 @@
+>    * a clear **rationale** for doing them? (*why* is it being changed)
+>    * a list of potential **points of contention**?
+>    * **links to relevant issues**, RFCs, MCPs, etc?
+> * Does the PR need a **regression test**?
+> * Does the change need to be covered by a **[major change proposal][mcp]**? Is it already covered?
+> * Would someone trying to understand the PR in a year's time be able to quickly **reconstruct what's going on**?
+> * Is the new code **properly documented**? Is existing documentation still up-to-date?
+> * Does the PR introduce a **regression** of any of the following:
+>   * error message quality
+>   * maintainability (e.g. complex code, no documentation, unsafe)
+>   * any specific target platforms
+>   * downstream tooling (e.g. linkers, debuggers)
+>   * compile times
+>   * memory usage
+>
+> #### Checklist for reviewers
+> * Am I the **right person to review** this PR:
+>   * Do I understand the code well enough?
+>   * Would I be able to spot non-obvious side effects?
+>   * Would I be able to fix a bug introduced by this PR?
+>   * Can I do the review in a timely fashion?
+>   * Do I feel pressure to quickly approve the PR for some reason?
+>   * Am I impartial enough?
+> * Before merging:
+>   * Is the **PR message still accurate**?
+>   * Is the **commit history clean** enough?
+
+
+
+## Guidance for dealing with common situations
+
+In most cases common sense is enough for deciding how to apply this policy.
+However, sometimes there are gray areas where it is not immediately clear how to proceed.
+This section lists a few common cases together with guidance on how to deal with them.
+
+* ### I don't think I am a good fit for reviewing - what now?
+  It is completely normal that you get assigned a PR (via highfive or otherwise) but don't feel comfortable reviewing it.
+  Here is what you can do, depending on the concrete case:
+
+  - If the change seems really big or contentious, consider asking for an MCP (see below).
+  - If you know just the right person for the review, assign them via `r? @<github-name>`.
+    It's polite to leave a comment asking them if they can take over --
+    but you don't have to make sure beforehand that they can actually do it.
+  - If you don't know the code well or already have too much on your plate,
+    ask highfive to roll the dice again via `r? @rust-lang/compiler-contributors`.
+
+  You can also always ask specific compiler team members for help finding a reviewer.
+
+* ### It is unclear if something constitutes a major change
+  Deciding if something is a "major change" requiring an [MCP][mcp] is not always straightforward.
+  The official guidelines are [here][whats-a-major-change].
+  When in doubt, err on side of treating something as a major change.
+  You can also nominate the PR for discussion in the compiler team's triage meeting by tagging it `I-nominated`.
+  If you nominate a PR please make sure to state a concrete question for the compiler team to discuss.
```

Comment:
@pnkfelix, you signaled that you wanted to discuss the wording of this paragraph.

Reply:
Not sure if this is still a concern.

Reply:
I'm going to mark this as resolved because it has changed a bit.